### PR TITLE
extend rule content structs because of different file lookup mechanism

### PR DESCRIPTION
**Pull request by Bee-lee**
# Description
Necessary for CCXDEV-4454

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## Testing steps
make test

### CODE REVIEW

**Review by tisnik — APPROVED** [review]
LGTM

File: types/content.go [hunk 1]
```diff
@@ -39,10 +40,13 @@ type RulePluginInfo struct {
 
 // RuleErrorKeyContent wraps content of a single error key.
 type RuleErrorKeyContent struct {
-	Generic   string           `json:"generic"`
-	Metadata  ErrorKeyMetadata `json:"metadata"`
-	TotalRisk int              `json:"total_risk"`
-	Reason    string           `json:"reason"`
+	Metadata   ErrorKeyMetadata `json:"metadata"`
+	TotalRisk  int              `json:"total_risk"`
```
Comment (epapbak) [inline types/content.go:44]: I'm not clear about this one. Why is it here?

**Review by juandspy — APPROVED** [review]

**Review by joselsegura — APPROVED** [review]

File: types/content.go (hunk 1 above)
Comment (Bee-lee) [reply types/content.go:44]: What do you mean? TotalRisk?

Comment (Bee-lee) [reply types/content.go:44]: it was always there, i just moved the field:)

**Pull request merged**